proposals/description/tip_115.ts

```diff
@@ -51,9 +51,60 @@ const tip_115: TemplatedProposalDescription = {
       method: 'acceptBeneficiary()',
       arguments: (addresses) => [],
       description: 'Accept beneficiary of rari infra tribe timelock'
+    },
+
+    //// Trigger new LBP of ETH
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'withdrawToSafeAddress(address,address,uint256,bool,bool)',
+      arguments: (addresses) => [
+        addresses.daiFixedPricePSM,
+        addresses.ethToDaiLBPSwapper,
+        '1000000000000000000000000',
+        false,
+        false
+      ],
+      description: 'Withdraw 1M DAI to LBP Swapper'
+    },
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'withdrawToSafeAddress(address,address,uint256,bool,bool)',
+      arguments: (addresses) => [
+        addresses.ethPSM,
+        addresses.ethToDaiLBPSwapper,
+        '10000000000000000000000',
+        false,
+        false
+      ],
+      description: 'Withdraw 10k ETH to LBP Swapper'
+    },
+    {
+      target: 'ethToDaiLBPSwapper',
+      values: '0',
+      method: 'swap()',
+      arguments: (addresses) => [],
+      description: 'Trigger new ETH LBP swap'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: (addresses) => [
+        '0x471cfe1a44bf1b786db7d7104d51e6728ed7b90a35394ad7cc424adf8ed16816', // SWAP_ADMIN_ROLE
```

Comment:
Nit, you can import `ethers` now and replace this with `ethers.utils.id('SWAP_ADMIN_ROLE)`